.rubocop.yml

```diff
@@ -17,18 +17,19 @@ Lint/ImplicitStringConcatenation:
     - 'lib/generators/sufia/**/*'
 
 Metrics/LineLength:
-  Max: 400
+  Enabled: false
 
 Metrics/AbcSize:
-  Max: 30
-  Exclude:
-    - 'app/controllers/concerns/sufia/admin/depositor_stats.rb'
-    - 'app/controllers/concerns/sufia/my_controller_behavior.rb'
-    - 'app/services/sufia/user_stat_importer.rb'
-    - 'lib/sufia/arkivo/metadata_munger.rb'
+  Enabled: false
 
 Metrics/MethodLength:
-  Max: 15
+  Enabled: false
```

Comment:
Wow, they shut off quite a lot. There's also a few things in our .rubocop.yml that are disabled right now that I think might be worth turning back on. I'll start a separate pull request for that, since it will probably need some manual cleanup (the Sufia folks are really lax about string styles and module nesting and some other basic stuff).

My personal vote would be to try to live within a reasonably sized method length (maybe 15-20 lines), and specifically add comments to disable the check on methods where we feel it's warranted to go longer. I think the benefit of doing things that way is that it forces exceptions to the style to be visible in pull requests and can help spark discussion in reviews about how we feel about the benefits and readability costs?